src/styles/blocks/icon.scss

```
@@ -0,0 +1,11 @@
+.icon {
+  width: 20px;
+  height: 13px;
+
+  &--menu {
+    background-image: url(../images/lines.png);
+    background-size: contain;
+    background-repeat: no-repeat; 
```

I think it's better to use short form
```suggestion
    background: contain no-repeat url(../images/lines.png);
```